bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/TestReplicationWorker.java

```diff
@@ -1262,6 +1268,9 @@ protected EnsemblePlacementPolicy initializeEnsemblePlacementPolicy(ClientConfig
             assertNull(stat1);
         });
 
+        assertTrue("NUM_NOT_ADHERING_PLACEMENT_LEDGERS_REPLICATED",
```

Comment:
Please add a new test to make sure the number is increased as expected.

Reply:
Good suggestion. I found a bug about numLedgersReplicated update. I have submit another PR #3654 to fix it. PTAL. I will update this PR when that PR is merged.

Reply:
+1